### PR TITLE
feat: Set default version for easy install to v13

### PR DESCRIPTION
**Pull request by abhishekbalam**
1. Updated default branch while cloning to `version-13`
2. ~~Updated Node version to 14 for v13 requirements~~
3. Updated test case for switch branch command too as it cant switch to a branch which isn't pulled

### CODE REVIEW

File: bench/tests/test_base.py [hunk 1]
```diff
@@ -13,7 +13,7 @@
 import bench.utils
 
 if sys.version_info.major == 2:
-	FRAPPE_BRANCH = "version-12"
+	FRAPPE_BRANCH = "version-13"
```
Comment (gavindsouza) [inline bench/tests/test_base.py]: Why setup v13 Frappe for PY2? They're incompatible.

File: bench/playbooks/roles/nodejs/defaults/main.yml [hunk 2]
```diff
@@ -1,3 +1,3 @@
 ---
-node_version: 12
+node_version: 14
```
Comment (gavindsouza) [inline bench/playbooks/roles/nodejs/defaults/main.yml]: Does this work with Frappe v12 installs?

File: bench/tests/test_base.py (hunk 1 above)
Comment (abhishekbalam) [reply bench/tests/test_base.py]: Right, will make changes

File: bench/playbooks/roles/nodejs/defaults/main.yml (hunk 2 above)
Comment (abhishekbalam) [reply bench/playbooks/roles/nodejs/defaults/main.yml]: Will test

Comment (abhishekbalam) [reply bench/playbooks/roles/nodejs/defaults/main.yml]: @gavindsouza 

I've left it as 12 as central runs v13 and v12 both with this.

File: bench/tests/test_base.py (hunk 1 above)
Comment (abhishekbalam) [reply bench/tests/test_base.py]: Changed develop to v13 instead

File: bench/tests/test_base.py [hunk 3]
```diff
@@ -15,7 +15,7 @@
 if sys.version_info.major == 2:
 	FRAPPE_BRANCH = "version-12"
 else:
-	FRAPPE_BRANCH = "develop"
+	FRAPPE_BRANCH = "version-13"
```
Comment (gavindsouza) [inline bench/tests/test_base.py]: This removes any way of knowing if bench is compatible with the latest Frappe develop. We might have to add more tests to ensure that.

What is the reason for doing this?

Comment (abhishekbalam) [reply bench/tests/test_base.py]: The idea was the latest stable version should be tested on priority over develop as that's what would be used most

If we want to write more tests for each version we can, but the tests will need to be updated for newer versions develop aside.

What do you suggest we do? Should I just keep it develop? @gavindsouza

Comment (gavindsouza) [reply bench/tests/test_base.py]: > The idea was the latest stable version should be tested on priority over develop as that's what would be used most


For testing version compatibility of Frappe with the Bench CLI, I think we still do test the relevant bits (bench creation, requirements resolution, assets building) in the Frappe CI. Anyway, Frappe and ERPNext CIs test this on push and branch merges daily, as compared to the relatively low frequency tests being run here.


> If we want to write more tests for each version we can, but the tests will need to be updated for newer versions develop aside.


Not sure if testing for each major version separately is the way to go. Frappe and Bench are fairly de-coupled in the cases these test suites are written for.


> What do you suggest we do? Should I just keep it develop? @gavindsouza


I'd stick with leaving this as `develop`. Don't have clear enough reasons to make any change.

**Comment by gavindsouza** [comment]
What's the status of this? @abhishekbalam

File: bench/tests/test_base.py (hunk 3 above)
Comment (abhishekbalam) [reply bench/tests/test_base.py]: Okay, makes sense

**Review by gavindsouza — APPROVED** [review]

**Pull request merged**